packages/tmlanguage-generator/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "devDependencies": {
     "@types/node": "~14.0.27",
-    "@types/plist": "~3.0.2"
+    "@types/plist": "~3.0.2",
+    "typescript": "~4.4.4"
```

Comment:
Not sure how this was building without that. Got some issue in building the docker file which in the end weren't related to that but this is still weird.